doc/source/topologies.rst

```diff
@@ -0,0 +1,137 @@
+Topologies
+==========
+
+
+Starting off
+-----------------
+Topology files are located in ``topologies`` in your streamparse project folder.
+There can be any number of topology files for your project in this directory.
+
+* topologies/my-topology.clj
+* topologies/my-other-topology.clj
+* topologies/my-third-topology.clj
+
+So on and so forth.
+
+Your topology file that describes the Directed Acyclic Graph (DAG) that will 
+be run in Storm.
+
+Looking at ``my-topology.clj``, we start off with importing the streamparse
+Clojure DSL functions
+
+.. code-block:: clojure
+
+    (ns my-topology
+      (:use     [streamparse.specs])
+      (:gen-class))
+
+Notice the ``my-topology`` matches the name of the file. We then have the
+import function.
+
+You could optionally avoid all of the streamparse-provided helper functions and 
+import your own functions or the Clojure DSL for Storm directly. 
+
+.. code-block:: clojure
+
+    (ns my-topology
+      (:use     [backtype.storm.clojure])
+      (:gen-class))
+
+In the next part of the file, we setup a topology definition, also named
+``my-topology`` (matching the ``ns`` line and filename). This definition is 
+actually a Clojure function that takes the topology options as a single map 
+argument. This function returns a list of 2 maps -- a spout map, and a bolt map.
+These two maps define the DAG that is your topology.
```

Comment:
The namespace has to match the name of the file, which has to match the name of the function, so yea, I think it does matter.